chat/chat/MessageViewController.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  ViewController.swift
+//  chat
+//
+//  Created by Gonzalo Romero on 06/11/2021.
+//
+
+import UIKit
+
+class MessageViewController: UIViewController {
+    
+    @IBOutlet weak var tableView: UITableView!
+    
+    var messages: [Message]? = [] {
```

Comment:
if you set [] as default value for message, maybe is better to remove the `?` from the type. It will simplify the code and will avoid to use force unwrap (!)